packages/grant-explorer/src/features/common/ProjectCard.tsx

```diff
@@ -97,7 +97,12 @@ export function ProjectCard(props: {
             </div>
           </CardDescription>
 
-          <Badge color="grey" rounded="3xl">
+          <Badge
+            color="grey"
+            rounded="3xl"
+            className="truncate max-w-full"
```

Comment:
Dunno what the implications are but what if we moved these classes to the component?